conda-forge.yml

```diff
@@ -1,8 +1,9 @@
+bot:
+  automerge: true
+  inspection: update-grayskull
```

Comment:
@FabianHofmann, I added the grayskull update feature here as I suggested to you.